<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingGeneral</class>
 <widget class="QWidget" name="SettingGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="m_closeToTray">
     <property name="text">
      <string>Close to system tray</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_runOnStart">
     <property name="text">
      <string>Run on start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_remRun">
     <property name="text">
      <string>Remember running</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>94</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
